sklearn/externals/_packaging/version.py

```diff
@@ -64,9 +64,17 @@
 
 def parse(version: str) -> Union["LegacyVersion", "Version"]:
     """
-    Parse the given version string and return either a :class:`Version` object
-    or a :class:`LegacyVersion` object depending on if the given version is
-    a valid PEP 440 version or a legacy version.
+    Parse the given version from a string to an appropriate class.
```

Comment:
PEP 257 recommends putting the first line on the same line as the `"""` marker itself.